### PR TITLE
Fix typo in message + remove zombie messages from Basic04

### PR DESCRIPTION
**Pull request by pnax**
## Purpose

Fix typos.

## Context

n/a

## Changes

...

## How to test this PR

...

### CODE REVIEW

**Review by marc-vanderwal — APPROVED** [review]
Looks good to me. But won’t merging this PR require #1167 to be rebased?

File: lib/Zonemaster/Engine/Test/Basic.pm [hunk 1]
```diff
@@ -154,11 +154,11 @@ Readonly my %TAG_DESCRIPTIONS => (
     },
     B04_MISSING_NS_RECORD => sub {
         __x    # BASIC:B04_MISSING_NS_RECORD
-          'Nameserver {ns} reponds to a NS query with no NS records in the answer section.', @_;
+          'Nameserver {ns} responds to a NS query with no NS records in the answer section.', @_;
```
Comment (matsduf) [inline lib/Zonemaster/Engine/Test/Basic.pm:157]: Test case Basic04 has been removed. These are just meaning less definitions of messages not used. It is better not to do any update not to trigger translation work.

File: lib/Zonemaster/Engine/Test/Basic.pm [hunk 2]
```diff
@@ -154,11 +154,11 @@ Readonly my %TAG_DESCRIPTIONS => (
...
     },
     B04_MISSING_SOA_RECORD => sub {
         __x    # BASIC:B04_MISSING_SOA_RECORD
-          'Nameserver {ns} reponds to a SOA query with no SOA records in the answer section.', @_;
+          'Nameserver {ns} responds to a SOA query with no SOA records in the answer section.', @_;
```
Comment (matsduf) [inline lib/Zonemaster/Engine/Test/Basic.pm:161]: As above.

**Comment by matsduf** [comment]
> Looks good to me. But won’t merging this PR require #1167 to be rebased?

#1167 does not have v2022.2 as milestone and is draft only. Is that meant to be included in v2022.2?

File: lib/Zonemaster/Engine/Test/Basic.pm (hunk 1 above)
Comment (pnax) [reply lib/Zonemaster/Engine/Test/Basic.pm]: Good point. I've removed them.

**Pull request merged**